compiler_gym/spaces/permutation.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import numpy as np
+
+from compiler_gym.spaces.scalar import Scalar
+from compiler_gym.spaces.sequence import Sequence
+
+
+class Permutation(Sequence):
+    def __init__(self, name: str, scalar_range: Scalar):
```

Comment:
The constructor needs a docstring and `   .. automethod:: __init__` in the rst

Reply:
Done.